catalog/MDCCatalog/MDCNodeListViewController.swift

```diff
@@ -454,6 +454,12 @@ extension MDCNodeListViewController {
 
       // TODO(featherless): Remove once
       // https://github.com/material-components/material-components-ios/issues/367 is resolved.
+      let dotsImage = MDCIcons.imageFor_ic_more_horiz()?.withRenderingMode(.alwaysTemplate)
```

Comment:
Pull this and its previous use in `MDCCatalogComponentsController.swift` to a shared method please.

Reply:
Done.